pom.xml

```diff
@@ -59,6 +59,7 @@
     <findbugs.maxRank>9</findbugs.maxRank>
     <hadoop.version>2.6.3</hadoop.version>
     <logback.version>1.1.3</logback.version>
+    <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
```

Comment:
This will redirect all logs to a file.  Correct?  I like making the tests less spammy.  However, I don't think it should be at the expense of not seeing error messages in console.  We could instead convert any `System.out.println` call to `log.debug`.  Test logging is configured to not print out any messags below ERROR.